src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Security/RequestHeaderConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Security;
+
+public static class RequestHeaderConstants
+{
+    /// <summary>
+    /// HTTP request header name for setting a valid Preview Token to access unpublished data.
```

Comment:
Minor, but we could shorten this down to: `Sets preview tokens for accessing unpublished data.`